docs/getting-started-guides/index.md

```diff
@@ -1,33 +1,24 @@
 ---
 ---
 
-Kubernetes can run on a range of platforms, from your laptop, to VMs on a cloud provider, to rack of
-bare metal servers.  The effort required to set up a cluster varies from running a single command to
-crafting your own customized cluster.  We'll guide you in picking a solution that fits for your needs.
+Kubernetes can run on a range of platforms, from your laptop, to VMs on a cloud provider, to rack of bare metal servers.  The effort required to set up a cluster varies from running a single command to crafting your own customized cluster.  We'll guide you in picking a solution that fits for your needs.
 
 If you just want to "kick the tires" on Kubernetes, we recommend the [local Docker-based](/docs/getting-started-guides/docker) solution.
 
-The local Docker-based solution is one of several [Local cluster](#local-machine-solutions) solutions
-that are quick to set up, but are limited to running on one machine.
+The local Docker-based solution is one of several [Local cluster](#local-machine-solutions) solutions that are quick to set up, but are limited to running on one machine.
 
-When you are ready to scale up to more machines and higher availability, a [Hosted](#hosted-solutions)
-solution is the easiest to create and maintain.
+When you are ready to scale up to more machines and higher availability, a [Hosted](#hosted-solutions)solution is the easiest to create and maintain.
 
-[Turn-key cloud solutions](#turn-key-cloud-solutions) require only a few commands to create
-and cover a wider range of cloud providers.
+[Turn-key cloud solutions](#turn-key-cloud-solutions) require only a few commands to create and cover a wider range of cloud providers.
 
-[Custom solutions](#custom-solutions) require more effort to setup but cover and even
-they vary from step-by-step instructions to general advice for setting up
-a Kubernetes cluster from scratch.
+[Custom solutions](#custom-solutions) require more effort to setup but cover and even they vary from step-by-step instructions to general advice for setting up a Kubernetes cluster from scratch.
```

Comment:
How about: 

[Custom solutions](#custom-solutions) vary from step-by-step instructions to general advice for setting up a Kubernetes cluster from scratch.